src/config.rs

```diff
@@ -1,7 +1,7 @@
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 
 const DEFAULT_APT_PORT: u16 = 4444;
-const DEFAULT_DATABASE_URL: &str = "mongodb://localhost:27107";
+const DEFAULT_DATABASE_URL: &str = "mongodb://localhost:27017";
```

Comment:
Make this an env and have a retrival funx

Reply:
This is just the fallback when there's no specified URL, we already have `DATABASE_URL` as an env variable

Reply:
(27017 is the default port, pretty sure 27107 was just a typo)